gradle.properties

```diff
@@ -1,6 +1,6 @@
 GROUP=com.squareup.misk
 
-VERSION_NAME=0.4.0-SNAPSHOT
+VERSION_NAME=0.9.0-SNAPSHOT
```

Comment:
This PR is mostly just version bump and whitespace fixes?

Reply:
you got it